lib/debug/local.rb

```diff
@@ -105,5 +109,240 @@ def after_fork_parent
       }
     end
   end
+
+  class UI_LocalTuiConsole < UI_LocalConsole
+    attr_reader :screen
+
+    def initialize(windows)
+      super()
+
+      @original_stdout = $stdout.dup
+      @original_stderr = $stderr.dup
+      @screen = Screen.new(width, height, windows, @original_stdout)
+
+      tui_console = self
+      original_stdout = @original_stdout
+      original_stderr = @original_stderr
+      Reline.output = original_stdout if defined?(Reline)
+
+      $stdout.define_singleton_method(:write) do |*args|
+        original_stdout.write(*args)
+        tui_console.puts(args.join)
+      end
```

Comment:
This is to listen to stdout's printing events and reflect the message on the TUI window as well. The patch will be cleared after `#deactivate` is called (see below).